exasol_script_languages_container_tool/lib/api/get_language_activation_builder.py

```diff
@@ -96,14 +62,43 @@ def get_language_activation_builder(
 
     if custom_aliases is None:
         custom_aliases = dict()
+    with open(Path(flavor_path) / "flavor_base" / "language_definition") as f:
+        lang_def_template = f.read()
+
+    template = Template(lang_def_template)
+    language_definition = template.render(
+        bucketfs_name="", bucket_name="", release_name="", path_in_bucket=""
+    )
+    languages_defs = language_definition.split(" ")
+    language_def_components_list = list()
+    for lang_def in languages_defs:
+        alias, url = parse_language_definition(lang_def)
+        if type(url) is LanguageDefinitionURL:
+            url.bucket_name = bucket_name
+            url.bucketfs_name = bucketfs_name
+            url.path_in_bucket = path_in_bucket
+            url.container_name = container_name
+        language_def_components_list.append(
+            LanguageDefinitionComponents(alias=alias, url=url)
+        )
+
+    if add_missing_builtin:
+        builtin_aliases = {slc_lang.name.upper(): slc_lang for slc_lang in SLCLanguage}
+        defined_aliases = [
```

Comment:
I probably would extract the addition of the built-in aliases into a function

Reply:
Done